Gemfile

```diff
@@ -96,6 +96,8 @@ gem 'pghero'
 # Job queue
 gem 'sidekiq'
 gem 'sidekiq-scheduler'
+# App metrics and statistics
+gem 'librato-rails', '~> 1'
```

Comment:
Docs specifically mention to use v1 of gem on Heroku:
https://github.com/librato/librato-rails#important-note-on-breaking-change